packages/shylu/shylu_dd/frosch/src/Tools/FROSch_SubdomainSolver_def.hpp

```diff
@@ -62,7 +62,7 @@ namespace FROSch {
     {
         FROSCH_TIMER_START(subdomainSolverTime,"SubdomainSolver::SubdomainSolver");
         FROSCH_ASSERT(!K_.is_null(),"FROSch::SubdomainSolver : ERROR: K_ is null.");
-        if (!ParameterList_->get("SolverType","Amesos").compare("Amesos")) {
+        if (!ParameterList_->get("SolverType","Amesos2").compare("Amesos")) {
```

Comment:
Why did you change the default?

Reply:
I had trouble with another application, but I am going to change it back.

Reply:
Did you already do that?

Reply:
I changed it back to Amesos.